articles/app-service/overview-managed-identity.md

```diff
@@ -2,7 +2,7 @@
 title: Managed identities overview - Azure App Service | Microsoft Docs
 description: Conceptual reference and setup guide for managed identities in Azure App Service and Azure Functions
 services: app-service
-author: mattchenderson
+author: mattchenderson, yevster
```

Comment:
Actually, I don't think you want to add the author tag here. You should still get all the notifications, etc. as a contributor, but I think we're planning to move these all toward content writers.

Reply:
Fixed